web/src/components/WordSurveyCell/WordSurveyCell.tsx

```diff
@@ -261,7 +257,7 @@ const WordImageSurveyComponent = ({
             <LikertScaleQuestionField
               name={GLOBAL_LIKERT_SCALE_QUESTION_GROUP_B}
               n={5}
-              question="Is the word cloud helps you review your evaluation?"
+              question="Is the word cloud help you review your evaluation?"
```

Comment:
I am not sure about this question, can we check with @hrynenkoo how to improve it ?

Reply:
Sure

Reply:
Maybe something like this: "Is this word cloud helpful when reviewing your initial evaluation?"